app/Repositories/Contracts/FollowRepository.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Contracts/FollowRepository.php</br>**Line**: 3</br>There must be one blank line after the namespace declaration